docs/getting_started.rst

```diff
@@ -0,0 +1,361 @@
+
+    .. code-block:: none
+
+        [p]load cog1 cog2 ...
+
+You can enable and disable everything you want, which means you can
+customize Red how you want!
+
+.. _getting-started-community-cogs:
+
+~~~~~~~~~~~~~~
+Community cogs
+~~~~~~~~~~~~~~
+
+There's an entire `community <https://discord.gg/red>`_ that contributes
+to Red. Those contibutors make additional cogs for you to use. You can
+download them using the downloader cog.
+
+You can start using the downloader cog by loading it: ``[p]load downloader``
+
+You can find cogs by searching on `cogs.red <https://cogs.red>`_. Find whatever you want,
+there are hundreds of cogs available!
+
+.. note:: ``cogs.red``, the website that list all of the cogs is not
+    ready for v3 yet. For now, you can refer to `this post
+    <https://cogboard.red/t/approved-repositories/210>`_.
+
+.. 26-cogs not available, let's use my repo :3
+
+Cogs comes with repositories. A repository is a container of cogs
+that you can install. Let's suppose you want to install the ``say``
+cog from the repository ``Laggrons-Dumb-Cogs``. You'll first need
+to install the repository.
+
+.. code-block:: none
+
+    [p]repo add Laggrons-Dumb-Cogs https://github.com/retke/Laggrons-Dumb-Cogs
+
+.. note:: You may need to specify a branch. If so, add its name after the link.
+
+Then you can add the cog
+
+.. code-block:: none
+
+    [p]cog install Laggrons-Dumb-Cogs say
+
+Now the cog is installed, but not loaded. You can load it using the ``[p]load``
+command we talked about before.
+
+.. _getting-started-permissions:
+
+-----------
+Permissions
+-----------
+
+Red works with different levels of permissions. Every cog defines
+the level of permission needed for a command.
+
+~~~~~~~~~
+Bot owner
+~~~~~~~~~
+
+The bot owner can access all commands on every guild. He can also use
+exclusive commands that can interact with the global settings
+or system files.
+
+If it *you* by default. If not set, use ``[p]set owner``
+
+~~~~~~~~~~~~
+Server owner
+~~~~~~~~~~~~
+
+The server owner can access all commands on his guild, except the global
+ones or those who can interact with system files (available for the
+bot owner).
+
+~~~~~~~~~~~~~
+Administrator
+~~~~~~~~~~~~~
+
+The administrator is defined by its roles. You can set an admin role
+with the ``[p]set adminrole`` command.
+
+For example, in the mod cog, an admin can use the ``[p]modset`` command
+which defines the cog settings.
+
+~~~~~~~~~
+Moderator
+~~~~~~~~~
+
+A moderator is a step above the average users. You can set a moderator
+role with the ``[p]set modrole`` command.
+
+For example, in the mod cog (again), a mod will be able to mute, kick and ban;
+but he won't be able to modify the cog settings with the ``[p]modset`` command.
+
+.. tip::
+    If you don't like the default permission settings for some commands or
+    if want to restrict a cog or a command to a channel/member, you can use
+    the permissions cog.
+
+.. _getting-started-hosting:
```

Comment:
Not sure why this section is here, this guide is for after the user installs the bot, so why should they be told how to host the bot *now*?

Reply:
Because the first question is usually "why do my bot disconnects when I shutdown my computer", and this ends up in a painful "how to setup VPS" tuto in #support

Reply:
I don't think this should be here, but I also don't know where this could go that would be better. If we has a FAQ this would be perfect for that, but I don't believe anything like that exists.